app/templates/render-tree.hbs

```diff
@@ -1,3 +1,7 @@
+{{#unless isClosed}}
+  {{warning-message close=(action "closeWarning")}}
+{{/unless}}
```

Comment:
Unfortunately we need to tell `x-list` the header's height (so that it can calculate its content height). The warning here adds to the header's height so the default height (31) no longer applies when the warning is opened. 

You can see the problem if you fill the render tree and scroll to the bottom while the warning is opened. You won't be able to see the last row.

The way to solve this is to do the following:

```js
// app/controllers/render-tree.js
headerHeight: computed('isWarningClosed', function() {
  return this.get('isWarningClosed') ? 31 : 56;
})
```

```js
// app/templates/render-tree.hbs
{{#x-list .... headerHeight=headerHeight}}`
```